python/hail/methods/statgen.py

```diff
@@ -204,6 +204,729 @@ def linreg(dataset, ys, x, covariates=[], root='linreg', block_size=16):
 
     return cleanup(MatrixTable(jm))
 
+@handle_py4j
+@typecheck(dataset=MatrixTable,
+           test=strlike,
+           y=Expression,
+           x=Expression,
+           covariates=listof(Expression),
+           root=strlike)
+def logreg(dataset, test, y, x, covariates=[], root='logreg'):
+    r"""Test each variant for association using logistic regression.
```

Comment:
The language should be made generic, apart from the describing the example. C.f.:

https://hail.is/docs/devel/methods/index.html?highlight=linreg#hail.methods.linreg

Reply:
Don't worry about updating LMM docs to generic language though, since right now is closely tied to kinship matrix and stat gen specific. We'll update the docs as that changes.